superbench/analyzer/file_handler.py

```diff
@@ -118,7 +118,7 @@ def output_excel_data_not_accept(writer, data_not_accept_df, rules):
 
             for rule in rules:
                 for metric in rules[rule]['metrics']:
-                    col_index = columns.index(metric)
+                    col_index = columns.index(metric) + 1
```

Comment:
What's the meaning for this hard code?

Reply:
it's a bug when applying the conditional format in excel, I will separate a pr for this